tests/test_cases/test_cocotb/test_handle.py

```diff
@@ -50,6 +50,20 @@ async def test_string_handle_takes_bytes(dut):
     assert val == b"bytes"
 
 
+@cocotb.test(skip=cocotb.SIM_NAME.lower().startswith("icarus"))
+async def test_string_ansi_color(dut):
+    await cocotb.triggers.Timer(10, 'ns')
+    dut.stream_in_string.value = b"[34mblue[49m[39m"
```

Comment:
Can you use python escape sequences here instead?
```suggestion
    dut.stream_in_string.value = b"\x1b[34mblue\x1b[49m\x1b[39m"
```

Reply:
Done.